scratch/list_prs.py

```
@@ -0,0 +1,12 @@
+import json
+import os
+
+file_path = r'C:\Users\siddh\.gemini\antigravity\brain\efae1060-aa30-4e5e-803c-5c3570d3dbf8\.system_generated\steps\515\output.txt'
+with open(file_path, 'r', encoding='utf-8') as f:
```

Ruff (0.15.12)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)